core/examples/luigi-example-js/public/views/home.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <link rel='stylesheet' href='/fiori-fundamentals/fiori-fundamentals.min.css'>
+  <style type="text/css">
+    .fd-panel {
+      padding: 15px;
+    }
+  </style>
+</head>
+<body>
+  <section class="fd-section">
+    <div class="fd-section__header">
+      <h1 class="fd-section__title">Home</h1>
+    </div>
+    <div class="fd-panel" id="init-text"></div>
+  </section>
+
+  <script type="text/javascript" src="/luigi-client/luigi-client.js"></script>
+  <script type="text/javascript">
+    LuigiClient.addContextUpdateListener(updatedContext => {
+      console.log('Luigi Client updated');
+      document.getElementById("init-text").textContent = 'Luigi Client updated.';
+    });
+    LuigiClient.addInitListener(initialContext => {
+      document.getElementById("init-text").textContent = 'Luigi Client initialized.' ;
+      console.log('Luigi Client initialized');
```

Comment:
```suggestion

```

Reply:
We already send the same message to the `<div class="fd-panel" id="init-text"></div>` so it is not needed in the console.